mms/export_model.py

```diff
@@ -180,7 +180,7 @@ def generate_manifest(symbol_file, params_file, service_file, signature_file, mo
     manifest["Model"]["Description"] = model_name
     manifest["Model"]["Model-Name"] = model_name
     manifest["Model"]["Model-Format"] = "MXNet-Symbolic"
-    manifest["Engine"]  = {"MXNet":0.12}
+    manifest["Engine"]  = {"MXNet":mx.__version__}
```

Comment:
hmm. So we're grabbing this off the locally installed MXNet - is this what we want to do? what happens if the user has an old version of MXNet?

Reply:
We are using installed(local) mxnet to do model save, I think it makes sense to put it here.

Reply:
for onnx models this might make sense, since we are creating the symbol and params file with current version of mxnet. but for user exporting (packaging) existing mxnet model, the current version might not match the actual model files, so we would be creating incorrect manifest. Can we parse this from the model files instead?

Reply:
Mxnet models does not record the version when saving models.
And actually in our exporter, we are using onnx converter to convert onnx to mxnet. At here(https://github.com/awslabs/mxnet-model-server/blob/master/mms/export_model.py#L194), we are using local mxnet to serialize the model from memory to disk. So I think the version is still correct.

Reply:
1. that path is only executed for onnx models, not for mxnet models
2. i think mxnet does record the version number. because i have seen errors like this at runtime:

src/nnvm/legacy_json_util.cc:185: Warning: loading symbol saved by MXNet version 1201 with lower version of MXNet v1200. May cause undefined behavior. Please update MXNet if you encounter any issue

Reply:
Agree with what you said. 
Not all the mxnet model has version, at least the models I tried from model zoo does not have it. But I do find some model has version attr. 
For now I will add the version number from symbol if it has, otherwise the local mxnet version.

Reply:
👍